examples/camera-app/linux/include/camera-device.h

```diff
@@ -40,9 +40,19 @@
 #define MICROPHONE_MIN_LEVEL (1)
 #define MICROPHONE_MAX_LEVEL (254)
 #define INVALID_SPKR_LEVEL (0)
+#define DEFAULT_PAN (0)
+#define DEFAULT_TILT (0)
+#define DEFAULT_ZOOM (1)
```

Comment:
Why defines for this rater than constexpr ?

Reply:
Changed to use the server cluster defined constants.